docfx_yaml/extension.py

```diff
@@ -141,6 +156,14 @@ def _create_datam(app, cls, module, name, _type, obj, lines=None):
         # Make relative
         path = path.replace('/', '', 1)
         start_line = inspect.getsourcelines(obj)[1]
+
+        # append relative path defined in conf.py (in case of "binding python" project)
+        try:
+            source_prefix  # does source_prefix exist in the current namespace
```

Comment:
azure-python-sdk may need this.  sphinx.ext.extlinks  extension using same way to append some prefix.
also we have exception to handle it,  so no impact to other users.